Mathlib/Data/List/Basic.lean

```diff
@@ -4160,37 +4160,37 @@ end ZipRight
 #noalign list.to_chunks_join
 #noalign list.to_chunks_length_le
 
-/-! ### all₂ -/
+/-! ### Forall -/
 
-section All₂
+section Forall
 
 variable {p q : α → Prop} {l : List α}
 
 @[simp]
-theorem all₂_cons (p : α → Prop) (x : α) : ∀ l : List α, All₂ p (x :: l) ↔ p x ∧ All₂ p l
+theorem Forall_cons (p : α → Prop) (x : α) : ∀ l : List α, Forall p (x :: l) ↔ p x ∧ Forall p l
```

Comment:
This is a theorem, so it should be lowercased as
```suggestion
theorem forall_cons (p : α → Prop) (x : α) : ∀ l : List α, Forall p (x :: l) ↔ p x ∧ Forall p l
```
Same throughout. Hopefully there are no name clashes...

Reply:
Ah, sorry for that. `forall_iff_forall` will look strange… It could be. `forall_iff_forall_mem` which would be consistent with other lemmas about `forall_mem`.

Or maybe the def should be `Forall\_1`?

Reply:
There seem to be no clashes, let’s see if CI agrees.